README.md

```diff
@@ -78,12 +146,153 @@ val records: DataFrame = sqlContext.redshiftFile(path, Seq("name", "age"))
 val records: DataFrame = sqlContext.redshiftFile(path, "name varchar(10) age integer")
 ```
 
-Migration Guide
----------------
+## Parameters
+
+The parameter map or <tt>OPTIONS</tt> provided in Spark SQL supports the following settings.
+
+<table>
+ <tr>
+    <th>Parameter</th>
+    <th>Required</th>
+    <th>Default</th>
+    <th>Notes</th>
+ </tr>
+
+ <tr>
+    <td><tt>redshifttable</tt></td>
+    <td>Yes</td>
+    <td>No default</td>
+    <td>The table to create or read from in Redshift</td>
```

Comment:
Not tested, but I expect not, unfortunately, as I wasn't expecting it in the various places I reference the table name string from parameters. I mentioned this in the discussion on #8 but couldn't find a nice way to enable users to either work with a whole table directly or push some work out to Redshift that they know it'll handle well, then work with the resulting data set in a DataFrame.

Does the JDBC data source support that? I guess I missed it. Maybe we could raise an issue to add support for this in a separate change? It would be a nice a feature.